### PR TITLE
EDU-968 Streamline linting and build

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-968

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export async function testWatch() {
+  await vitest.watch();
+}
+
+export const build = done => done();
+
+export const verify = gulp.series(lint, test, build);
+
+export default verify;
```
Comment (adrianaluchian) [inline gulpfile.js:29]: Tiny thing, but would be nice to mention the gulp tasks in the README maybe.

File: src/s3-helper.js [hunk 2]
```diff
@@ -67,7 +66,6 @@ export async function deleteAllObjects(s3, bucketName) {
   const oldObjects = await listAllObjects({ s3, bucketName });
   for (const obj of oldObjects) {
     console.log(`Deleting object ${obj.Key}`);
-    // eslint-disable-next-line no-await-in-loop
```
Comment (adrianaluchian) [inline src/s3-helper.js:70]: Nice! We've always been making an exception for this one as it was always an "informed decision" to ignore this error.

**Review by adrianaluchian — APPROVED** [review]

File: src/s3-helper.js (hunk 2 above)
Comment (ahelmberger) [reply src/s3-helper.js:70]: From now on we can be as uninformed as we want :rofl:

Comment (adrianaluchian) [reply src/s3-helper.js:70]: 😂 Oblivion is bliss

**Pull request merged**